riscv-regs/Cargo.toml

```diff
@@ -7,5 +7,6 @@ edition = "2021"
 
 [dependencies]
 tock-registers = { version = "0.7" }
+riscv-decode = "0.2"
```

Comment:
spelling in commit message for instruction decoding: missiung

Reply:
Done.